src/pages/AlertsPage.tsx

```diff
@@ -1,3 +1,62 @@
+import { Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useMemo } from 'react';
+
+import { AlertsContainer } from '../components/Alerts/AlertsContainer';
+import { Loader } from '../components/Common/Loader';
+import { getUnlabelledLatestSequences } from '../services/alerts';
+import { getCameraList } from '../services/camera';
+import { type AlertType, convertSequencesToAlerts } from '../utils/alerts';
+import { useTranslationPrefix } from '../utils/useTranslationPrefix';
+
 export const AlertsPage = () => {
-  return <div>AlertsPage</div>;
+  const { t } = useTranslationPrefix('alerts');
+
+  const {
+    isPending: isPendingSequences,
+    isError: isErrorSequences,
+    data: sequenceList,
+    isSuccess: isSuccessSequences,
+  } = useQuery({
+    queryKey: ['unlabelledSequences'],
+    queryFn: getUnlabelledLatestSequences,
+  });
+
+  const {
+    isPending: isPendingCameras,
+    isError: isErrorCameras,
+    data: cameraList,
+    isSuccess: isSuccessCameras,
+  } = useQuery({
+    queryKey: ['cameras'],
+    queryFn: getCameraList,
+  });
+
+  const alerts: AlertType[] = useMemo(
+    () => convertSequencesToAlerts(sequenceList ?? [], cameraList ?? []),
+    [sequenceList, cameraList]
+  );
+
+  const isPending = isPendingSequences || isPendingCameras;
+  const isError = isErrorSequences || isErrorCameras;
+  const isSuccess = isSuccessSequences && isSuccessCameras;
+
+  return (
+    <>
+      {isPending && <Loader />}
+      {isError && (
+        <Typography variant="body2">
+          {t('errorFetchSequencesMessage')}
+        </Typography>
+      )}
+      {isSuccess && (
+        <>
+          {alerts.length == 0 && (
+            <Typography variant="body2">{t('noAlertsMessage')}</Typography>
+          )}
+          {alerts.length != 0 && <AlertsContainer alerts={alerts} />}
```

Comment:
I like that pattern where we get all the necessary data in the Page component and pass it down. 
It reduces the number of loading state and everything loads at once AND in some cases, it makes it easier to reuse/test component in isolation (like storybook) since we just pass data as props

Reply:
not for this PR but I would also do that for the DashboardPage then to stay consistent

Reply:
As I have two PR based on this one, I'm a bit shy about this change (conflicts coming). Would it be ok if I create a issue for for both page (dashboard and alert) and I fix in another PR ?